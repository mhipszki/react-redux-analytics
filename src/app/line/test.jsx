import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';

import Line from './';

test('styles an h2 element', () => {
  const component = shallow(<Line>text</Line>);
  expect(component).toMatchSnapshot();
});
