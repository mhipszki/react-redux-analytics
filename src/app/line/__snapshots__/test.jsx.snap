// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`styles an h2 element 1`] = `
.c0 {
  font-size: 1.5rem;
  font-weight: normal;
  margin-top: 0;
  color: #777;
}

<Line
  className="c0"
  isBold={false}
  text="text"
>
  text
</Line>
`;
